rad-cli/src/bin/auth.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2021 The Radicle Link Contributors
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+extern crate rad_cli;
+
+use structopt::StructOpt;
+
+use rad_clib::keys::ssh::SshAuthSock;
+
+use rad_cli::{keys, person, proc::some_or_exit, profile, tui};
+
+#[derive(Debug, StructOpt)]
+pub struct Args {
+    #[structopt(short, long)]
+    pub add: bool,
+}
```

Comment:
I'm ok with using structopt for now as long as we don't invest too heavily in it, but eventually I think I'd like all CLIs to use `lexopt` which we use in some of the other crates. The reason is that declarative configuration often isn't flexible enough to encode all the logic and you end up having to have a second phase of argument processing after the initial one.

Reply:
Ok, thanks for the explanation. I switched to `lexopt` already.